components/builder-worker/build.rs

```diff
@@ -39,3 +41,23 @@ fn write_docker_pkg_ident() {
     };
     util::write_out_dir_file("DOCKER_PKG_IDENT", ident);
 }
+
+fn write_kubernetes_exporter_pkg_ident() {
+    let ident = match env::var("PLAN_KUBERNETES_EXPORTER_PKG_IDENT") {
+        // Use the value provided by the build system if present
+        Ok(ident) => ident,
+        // Use the latest installed package as a default for development
+        _ => String::from("core/hab-pkg-export-kubernetes"),
+    };
+    util::write_out_dir_file("KUBERNETES_EXPORTER_PKG_IDENT", ident);
+}
+
+fn write_kubectl_pkg_ident() {
+    let ident = match env::var("PLAN_KUBECTL_PKG_IDENT") {
+        // Use the value provided by the build system if present
+        Ok(ident) => ident,
+        // Use the latest installed package as a default for development
+        _ => String::from("core/kubernetes"),
```

Comment:
I use `core/kubernetes` atm @schu 
Will that work?

Reply:
I should work as 1.8.3 has been promoted to stable (https://bldr.habitat.sh/#/pkgs/core/kubernetes) and 1.7.5 was there already.

I've also submitted a [kubectl plan](https://github.com/habitat-sh/core-plans/pull/1058), pending.

Reply:
Thanks 👍

Reply:
`core/kubectl` is now available.